packages/app/src/routers/ViewRouter/ViewRouter.tsx

```diff
@@ -36,25 +37,37 @@ const useThemeWatcher = () => {
   return theme;
 };
 export interface ViewProps extends RouteComponentProps {
+  accessToken?: string;
   projectId?: string;
   iModelId?: string;
+  versionId?: string;
 }
 const View = (props: ViewProps) => {
   (window as any).ITWIN_VIEWER_HOME = window.location.origin;
   const config = useConfig();
   const buddiRegion = config.buddi?.region
     ? parseInt(config.buddi.region)
     : undefined;
-  return (
+  const {
+    results: { namedVersion: fetchedVersion },
+    state,
+  } = useApiData<{ namedVersion: { changesetId: string } }>({
+    accessToken: props.versionId ? props.accessToken : undefined,
+    url: `https://api.bentley.com/imodels/${props.iModelId}/namedversions/${props.versionId}`,
+  });
+  const theme = useThemeWatcher();
+  const changesetId = props.versionId ? fetchedVersion?.changesetId : undefined;
+  return state || !props.versionId ? (
```

Comment:
if state is error, the viewer is still rendered, is that intended?

Reply:
Yes, it falls back to no latest Checkpoint. Still going happy path here.